src/IterableSchema.js

```diff
@@ -18,4 +18,9 @@ export default class ArraySchema {
   getItemSchema() {
     return this._itemSchema;
   }
+
+  mappedBy(foreignKey) {
+    this._mappedBy = foreignKey;
```

Comment:
It seems a bit inconsistent that the function uses a fluent API (as in, returns this while mutating the original object) while there is not a single such precedent in the existing API. What does `schema.mappedBy('a').mappedBy('b')` mean? Fluent APIs are confusing when they are mutating, and I think we should have cloned the schema here instead.